afl-fuzz.c

```diff
@@ -3440,7 +3449,8 @@ static void write_stats_file(double bitmap_cvg, double stability, double eps) {
              "afl_banner        : %s\n"
              "afl_version       : " VERSION "\n"
              "target_mode       : %s%s%s%s%s%s%s\n"
-             "command_line      : %s\n",
+             "command_line      : %s\n"
+             "slowest_exec_ms   : %llu\n",
```

Comment:
Are we naming/making this `slowest_exec_ms` instead of libFuzzer's `slowest_unit_time_sec` because the other AFL time stats are reported in ms and because AFL's timeout is usually < 1 second?

Reply:
I know that libFuzzer measures slow units in seconds. But I think that since we gather the times in ms granularity (even μs) it's better to keep ms, because seconds seem a bit too coarse in a fuzzing context.
Talking about the name, if we decide to keep it in ms, I think it's better to mention it so that people don't get confused. I don't have an opinion for `unit` or `exec`.
Can we continue to discuss this is #23 ?